### PR TITLE
[#82] [UI] As a signed-in user, I can see the User Menu on the left home Home Screen when tap the avatar

### PR DESCRIPTION
**Pull request by ducbm051291**
- #82 

## What happened

- Allow user to see Home screen's User Menu by tapping the avatar
 
## Insight

- Create HomeUserMenuView
- Implement HomeUserMenuView in HomeView
 
## Proof Of Work

https://user-images.githubusercontent.com/24598204/207829189-55feb776-23e1-47eb-9651-bc5660f99307.mp4

### CODE REVIEW

File: Surveys/Sources/Presentation/Modules/HomeView/HomeViewModel.swift [hunk 1]
```diff
@@ -13,6 +13,7 @@ import Resolver
 final class HomeViewModel: ObservableObject {
 
     @Published var state: State = .loading
+    @Published var version: String = Bundle.main.version
```
Comment (blyscuit) [inline Surveys/Sources/Presentation/Modules/HomeView/HomeViewModel.swift:16]: I think `Bundle` should be dependency injected 
```
init(bundle: Bundle) {
_version = Bundle.version
}
```

File: Surveys/Sources/Presentation/Modules/HomeView/View/HomeUserMenuView.swift [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                    Text(Localize.commonCancelText()),
+                    action: { isConfirmAlertVisible = false }
+                )
+            )
+        }
+        .edgesIgnoringSafeArea(.all)
+    }
+
+    private func setUpAvatarButton() -> some View {
+        Button(action: {}, label: {
```
Comment (blyscuit) [inline Surveys/Sources/Presentation/Modules/HomeView/View/HomeUserMenuView.swift:48]: This is not a button in the design? 🤔

**Review by Shayokh144 — APPROVED** [review]

File: Surveys/Sources/Presentation/Modules/HomeView/View/HomeUserMenuView.swift (hunk 2 above)
Comment (ducbm051291) [reply Surveys/Sources/Presentation/Modules/HomeView/View/HomeUserMenuView.swift:48]: Woa, good catch

File: Surveys/Sources/Presentation/Modules/HomeView/HomeViewModel.swift [hunk 3]
```diff
@@ -13,6 +13,11 @@ import Resolver
 final class HomeViewModel: ObservableObject {
 
     @Published var state: State = .loading
+    @Published var version: String = .empty
+
+    init(bundle: Bundle) {
+        version = Bundle.main.version
```
Comment (blyscuit) [inline Surveys/Sources/Presentation/Modules/HomeView/HomeViewModel.swift:19]:
```suggestion
        version = bundle.version
```

Comment (ducbm051291) [reply Surveys/Sources/Presentation/Modules/HomeView/HomeViewModel.swift]: Wow, how could I miss this 😢

**Review by blyscuit — APPROVED** [review]

**Pull request merged**